src/Features/Core/Portable/Copilot/ICopilotCodeAnalysisService.cs

```diff
@@ -84,4 +85,12 @@ internal interface ICopilotCodeAnalysisService : ILanguageService
     /// </summary>
     /// <param name="proposal">The documentation comment that has been broken down into its individual pieces.</param>
     Task<(Dictionary<string, string>? responseDictionary, bool isQuotaExceeded)> GetDocumentationCommentAsync(DocumentationCommentProposal proposal, CancellationToken cancellationToken);
+
+    /// <summary>
+    /// Method to retrieve the implementation of a method for a given <paramref name="proposal"/>
+    /// </summary>
+    /// <param name="proposal"></param>
+    /// <param name="cancellationToken"></param>
+    /// <returns></returns>
+    Task<(Dictionary<string, string>? responseDictionary, bool isQuotaExceeded)> GetMethodImplementationAsync(MethodImplementationProposal proposal, CancellationToken cancellationToken);
```

Comment:
this should be called ImplementNotImplementedExceptionAsync.

Reply:
This was never changed but the comment was resolved.

Reply:
commented elsewhere, all the resolved comments should be properly addressed when you view the latest version